### PR TITLE
checking user02 branch

### PR DESCRIPTION
**Pull request by ghost**
check if all code is good

### CODE REVIEW

File: user02.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+this is user02
```
Comment (MHRSRoni) [inline user02.txt:1]: this line is not necessary now

File: user02.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+this is user02 
+changin when in pull
```
Comment (ghost) [inline user02.txt:2]: this is ok

File: user02.txt (hunk 1 above)
Comment (ghost) [inline user02.txt:1]: should delete this code?

Comment (ghost) [reply user02.txt:1]: are you sure?

Comment (ghost) [reply user02.txt:1]: ok

**Pull request merged**